wiz-broker-v2/.helmignore

```diff
@@ -0,0 +1,23 @@
+# Patterns to ignore when building packages.
```

Comment:
I'm not sure we really need a `v2` dedicated chart here,

You could just apply your changes to the existing chart and bump its version.

Consult with @barmagnezi regarding this change as well